package.json

```diff
@@ -37,10 +40,14 @@
         "adobe"
     ],
     "scripts": {
-        "test": "nyc mocha --recursive --exit",
+        "test": "nyc mocha --recursive --exit ./test",
+        "e2e": "mocha --recursive --exit ./e2e",
         "posttest": "eslint . && license-checker --summary",
         "report-coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
         "beautify": "eslint . --fix",
         "semantic-release": "semantic-release"
+    },
+    "engines": {
+        "node": ">=18"
```

Comment:
node.js.yml refers to node-version: [14.17]. Would this have any impact?

Reply:
good catch. this could be a left-over. will review.